Backend/Backend.WebServices/Controllers/EnterpriseController.cs

```diff
@@ -51,11 +51,26 @@ public IEnumerable<EnterpriseDTO> Get()
         /// Given an Id (and possibly a password), get details of a specific enterprise
         /// </summary>
         /// <param name="id">ID of the enterprise whose details will be returned</param>
-        /// <returns>A string representing an enterprise</returns>
-        public string Get(int id)
+        /// <param name="password">Password of the enterprise</param>
+        /// <returns>An EnterpriseDTO with the specified ID</returns>
+        public EnterpriseDTO Get(int id, string password = null)
         {
             ThrowExceptionIfNotHttps();
-            return "ForestedgeCommunityGarden";
+            var enterprises = _context.Enterprises.ToList();
```

Comment:
It's probably not a good idea to do a ToList() on the whole enterprises context - this will immediately fire off a request to the database that gets all the enterprises, when for performance reasons we should only get one.
Also, this will only include the items in the Enterprise table, when we want to do joins to the Participant, Place, and Media Items in the same query (using the db foreign keys).
The actual query to get the single enterprise should therefore be something like:
```
var enterprise = _context.Enterprises
    .Where(x => x.Id == id)
    .Include(x => x.Participants.Select(p => p.Places.Select(pl => pl.MediaItems.Select(m => m.MediaItemType))))
    .SingleOrDefault();
if (enterprise == null)
{
    //throw appropriate exception - the id they've sent us isn't a valid enterprise ID!
}
```
All the stuff in the include line will be returned with the enterprise (ie the enterprise will have all its participants, places, media items and media item types populated in the object graph.